src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static String line;
+    private static String tmp = "";
+    private static String[] dataFromFile;
```

Comment:
There are static variables, that are not related to the object.
It means they are shared and if multiple threads will be executed, they all will work with this data.
So, this data are not isolated.
Each creation of a report should be performed in the method with the corresponding method and local variables.

As static, you should store only constants (operations, indexes of operations in CSV file)